[Update] Manual Compression (#906)
timescale/docs: timescaledb/how-to-guides/compression/manually-compress-chunks.md

## Manually compress chunks in a single command
Alternatively, you can select the chunks and compress them in a single command
by using the output of the `show_chunks` command to compress each one. For
example, use the following command to compress chunks if it is not already
compressed in the given time interval:

> :
> 

Reviewer: ```suggestion
```

## Manually compress chunks in a single command
Alternatively, you can select the chunks and compress them in a single command
by using the output of the `show_chunks` command to compress each one. For

> example, use the following command to compress chunks between one and three weeks old 

Reviewer: ```suggestion
example, use this command to compress chunks between one and three weeks old 
```

this:

```sql

> SELECT compress_chunk(i, if_not_compressed => true) from show_chunks('example', now() - interval '1 week', now() - interval '3 weeks') i;

Author: Just added an explanation.
